keylime/cloud_verifier_common.py

```diff
@@ -207,6 +207,7 @@ def process_quote_response(agent, json_response):
         ima_keyring,
         mb_measurement_list,
         {})
+
```

Comment:
Is this blank line to make pylint happy? If so, it should be a separate commit as it's not related to this change.

Reply:
No, I think it was an artifact of an application of a patch. I can remove if needed.

Reply:
Yes please. There's no reason to include files in the git history that weren't functionally changed.